core/__tests__/tasks/appRefreshQuery/checkAll.ts

```diff
@@ -0,0 +1,92 @@
+import { helper } from "@grouparoo/spec-helper";
+import { api, task, specHelper } from "actionhero";
+import { classToInvokable } from "sequelize/types/lib/utils";
+import {
+  Source,
+  Schedule,
+  App,
+  Run,
+  GrouparooModel,
+  AppRefreshQuery,
+} from "../../../src";
+
+describe("tasks/appRefreshQueries:checkAll", () => {
+  helper.grouparooTestServer({ truncate: true, enableTestPlugin: true });
+  beforeEach(async () => await api.resque.queue.connection.redis.flushdb());
+
+  beforeAll(async () => {});
+
+  describe("appRefreshQueries:checkAll", () => {
+    let model: GrouparooModel;
+    let app: App;
+    let source: Source;
+    let schedule: Schedule;
+    let appRefreshQuery: AppRefreshQuery;
+
+    beforeAll(async () => {
+      ({ model } = await helper.factories.properties());
+      app = await helper.factories.app(model);
+      source = await helper.factories.source(app);
+      await source.setOptions({ table: "test table" });
+      await source.setMapping({ id: "userId" });
+      await source.update({ state: "ready" });
+
+      schedule = await helper.factories.schedule(source);
+      await schedule.update({ state: "ready" });
+
+      appRefreshQuery = new AppRefreshQuery({
+        appId: app.id,
+        refreshQuery: "SELECT 'hi' AS name;",
+        state: "ready",
+      });
+      await appRefreshQuery.save();
+    });
+
+    test("can be enqueued and run", async () => {
+      await task.enqueue("appRefreshQueries:check", {}); //does not throw
+
+      await specHelper.runTask("appRefreshQueries:check", {});
+      const found = await specHelper.findEnqueuedTasks(
+        "appRefreshQueries:check"
+      );
```

Comment:
These tests may be testing the wrong task (`appRefreshQueries:checkAll`) .  These tests also probably don't care that any runs were enqueued, but that the next task, `appRefreshQueries:check` was.

Reply:
This might be a hint that the names of these tasks aren't quite descriptive enough... `:check` vs `:query` might help?